common/cloudant_base.go

```diff
@@ -195,3 +200,13 @@ func GetAuthenticatorFromEnvironment(credentialKey string) (core.Authenticator,
 
 	return core.GetAuthenticatorFromEnvironment(credentialKey)
 }
+
+// buildUserAgent builds the user agent string.
+func buildUserAgent() string {
+	return fmt.Sprintf("cloudant-go-sdk/%s(%s)", Version, getSystemInfo())
+}
+
+// getSystemInfo returns the system information.
+func getSystemInfo() string {
+	return fmt.Sprintf("%s;%s", runtime.Version(), runtime.GOOS)
```

Comment:
You should chop runtime.Version() to 8 characters and runtime.GOOS to 3 characters.

Reply:
I would not suggest to cut down the `runtime.Version()`, it can help a lot in special cases.
For `runtime.GOOS` why 3 characters? Interestingly it is enough to differentiate the OS types. But do we need to save characters such strictly in the headers?

Reply:
I would suggest to limit the header conent, when there is more specific detail about the bonus fields.

Reply:
ok, shortened in https://github.com/IBM/cloudant-go-sdk/pull/250/commits/de04d55e493738e119d92e8af40d10e08322bf6c

let's decide on arch on channel, can amend it in another commit if we'll agree to include it